handlers.go

```diff
@@ -107,13 +118,15 @@ func (sf *Sunfish) GetFile(w http.ResponseWriter, r *http.Request) {
 	// Query and find by one id
 	err := sf.DB.C("siafiles").FindId(bson.ObjectIdHex(id)).One(&siafile)
 	if err != nil {
-		panic(err)
+		sf.logger.Println("ERROR: Could not find Siafile: ", bson.ObjectIdHex(id))
```

Comment:
I disagree. Logging errors like this can show us if people are doing things like searching for hidden files.